web/src/gear_menu.ts

```diff
@@ -1,5 +1,7 @@
 import $ from "jquery";
+import type {Instance as TippyInstance} from "tippy.js";
```

Comment:
```suggestion
import type * as tippy from "tippy.js";
```

We always use namespace imports for Tippy (#30265).